scalafix/rules/src/main/scala/kyo/rules/KyoUpdateToV1_0.scala

```diff
@@ -44,6 +44,13 @@ class KyoUpdateToV1_0 extends SemanticRule("KyoUpdateToV1_0") {
 
             case resource @ q"Resource" if resource matches "kyo.Resource" =>
                 Patch.replaceTree(resource, "Scope")
+
+            // Async combinator
+            case fork @ q"$eff.fork" =>
+                Patch.replaceTree(fork, s"$eff.forkUnscoped")
+
+            case forkScoped @ q"$eff.forkScoped" =>
+                Patch.replaceTree(forkScoped, s"$eff.fork")
```

Comment:
Nice! I need to alter a bit the setup to test from 1.0-RC1 to main. (I will update in another PR)

Reply:
Yeah I was considering adding new rules just for RC1 -> 1.0 but would get a little complicated. Should we have separate rules for each RC?